internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -159,11 +162,15 @@ func (opts *Opts) Run() error {
 		return err
 	}
 
-	fmt.Printf(`We are deploying %s...`, opts.ClusterName)
 	if err := opts.createCluster(); err != nil {
+		if strings.Contains(strings.ToLower(err.Error()), freeClusterAlreadyExists) {
```

Comment:
string comparison for errors is a bad practice, please use [go1.13 error handling](https://go.dev/blog/go1.13-errors), there are a lot of [examples](https://github.com/mongodb/mongodb-atlas-cli/search?q=errors.As&type=code) across the cli on how to do this properly